packages/preview/university-assignment/0.1.0/typst.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "university-assignment"
```

Comment:
The package name is too generic and descriptive. Can you please change it to something more unique and distinctive?